forc-plugins/forc-client/src/cmd/call.rs

```diff
@@ -0,0 +1,141 @@
+use crate::NodeTarget;
+use clap::Parser;
+use either::Either;
+use fuel_crypto::SecretKey;
+use fuels_core::types::ContractId;
+use std::{path::PathBuf, str::FromStr};
+use url::Url;
+
+pub use forc::cli::shared::{BuildOutput, BuildProfile, Minify, Pkg, Print};
+pub use forc_tx::{Gas, Maturity};
+
+forc_util::cli_examples! {
+    super::Command {
+        [ Call a contract with function parameters => "forc call <CONTRACT_ID> <FUNCTION_SIGNATURE> <ARGS>" ]
```

Comment:
i wonder if being a bit more explicit on what these calls look like might be easier for people to reference? I could go either but maybe the above could be 

```
[ Call a simple contract function => "forc call 0x123..abc add 1 2" ]
```

thoughts?

Reply:
agreed; makes sense 👍